doc/userguide/upgrade.rst

```diff
@@ -30,6 +30,43 @@ by the ones Suricata supplies.
 Major updates include new features, new default settings and often also
 remove features.
 
+Suricata Configuration Defaults
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
Should this be changed to or include something about the idea of **built-in** defaults vs. **yaml** defaults?